### ISSUE TITLE
[Documentation] Title vs Subject

### ISSUE THREAD

**Issue by a-koka** [react-native-share/react-native-share#433]
It looks like `subject` is used for email and the `title` in other cases. Some detailed documentation would be great so users understand what is used, where, and when.

The behavior seems a little inconsistent from my testing when using both `title` and `subject`. The Android Mail app will append `title` to the body thus accounting for a repeated `subject` but on iOS this works as expected. Also, one field is ignored in some cases and both in others.

I realize that this is quite dependent upon the app being used to share but just wondering if we can provide more information for a better understanding?

**Comment by esutton** [comment]
[iOS] Behavior is different depending on destination app selected.

- Apple mail app - 'subject' and 'body' appear as expected.
   - Even HTML body text is working.  Excellent!
- SMS - Only 'body' appears
  - Multiple email attachments work.  Very good.
- **Gmail** and **Outlook** mail app 
  - 'body' replaces the email subject
  - 'title' does not appear.
  - Multiple email attachments work.  Very good.


````
      const shareOptions = {
        subject: 'subject',
        title: 'title',
        message: 'body',
        social: Share.Social.EMAIL,
        failOnCancel: false,
        urls,
      };
````

**Comment by UdithaKasun** [comment]
Any update on this ? I am also facing this issue with Gmail and Outlook client on iOS with react-native-share 1.1.3.

**Comment by techniqs** [comment]
I tried to share a PDF with Outlook (iOS), subject & title are completely ignored, message is pasted into the subject and body, file attachment works. I tried to use activityItemSources to maybe access subject and recipient-mail-address but couldn't figure it out.
Also: even when the default mail in settings is set to outlook, i can't use Share.single with social set to Share.Social.EMAIL.
Current Setup: `"react-native": "~0.63.3",
"react-native-share": "4.1.0",`
I hope either this issue gets addressed and fixed or there can be some workaround provided eg. to use the `activityItemSources.`
